src/pages/oncall-me/oncall-me.ts

```diff
@@ -46,6 +47,14 @@ export class OncallMePage {
     });
   }
 
+  getSmsUrl(smsNumber) {
+    var smslink = smsNumber.replace(/ /g, '');
+    smslink = smslink.replace(/-/g, '');
+    smslink.replace(/\+/, '');
+    smslink = 'sms:' + smslink;
+    return this.sanitizer.bypassSecurityTrustUrl(smslink);
```

Comment:
I think this opens an XSS vulnerability, since users have the ability to put basically anything into their phone number field depending on how contacts are set up (including arbitrary non-numeric text). To be safe, I would strip all non-numeric values in the SMS number.